tests/test_csc.py

```diff
@@ -78,7 +78,7 @@ async def test_lifecycle_behavior(self):
             self.telemetry_directory = self.csc.telemetry_directory
             self.assertFalse(self.csc.simulated_telemetry_loop_task.done())
             sim_files = len(list(os.listdir(self.csc.telemetry_directory)))
-            self.assertEqual(sim_files, 2)
+            self.assertEqual(sim_files, 1)
```

Comment:
Why do you remove one of the two files and not the other? What is the significance of having this file? Would it make sense to delete it at the end of the test?

Reply:
At the start of ENABLED state, the DSM simulator writes a configuration file and then starts writing data files for telemetry. Deleting the data file after sending the telemetry topic is why the test number got decremented as the data file is no longer there but the configuration file still is. The config file is only written on entry in to ENABLED, so unless one cycles the DSM LOTS of times, clean up isn't a big problem.

Reply:
> Also you check that the file exists, but I don't see any code that checks the _contents_ of the file. Is it practical to add that?

The contents are checked via the events that come out from the simulation loop since the file contents are stuffed into those events and sent. The telemetry file data are randomly generated so testing the content really isn't feasible.